template-for-repository/proofs/run-cbmc-proofs.py

```diff
@@ -234,8 +247,23 @@ async def main():
     proof_root = pathlib.Path(__file__).resolve().parent
     litani = get_litani_path(proof_root)
 
+    litani_caps = get_litani_capabilities(litani)
```

Comment:
Is this capabilities file something that is shipped with litani or is it something the start kit user is supposed to construct?

Reply:
It's not a file, this method is running `litani print-capabilities`, which makes Litani print out a JSON list of the features that it supports. This is because we don't know which version of Litani is going to be used with the starter kit, so the starter kit has to test whether Litani has certain features before using them.
```
> ~/code/litani/litani print-capabilities
[
  "atomic_report_update",
  "report_expire",
  "dir_lock_api",
  "dir_lock_api_v2",
  "outcome_table",
  "output_directory_flags",
  "pools"
]
```